docs/fsharp/language-reference/compiler-messages/fs0003.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Compiler Error FS0003"
+ms.date: 12/23/2019
+f1_keywords:
+  - "FS0003"
+helpviewer_keywords:
+  - "FS0003"
+---
+
+# FS0003: This value is not a function and cannot be applied
+
+This error is given when a parameter is given to a value that cannot accept parameters.  This is often the case when attempting to use an indexer into a collection like a list, array, or IEnumerable.
+
+The following code demonstrates one way the error can be seen:
+
+[!code-fsharp[FS0003-pass-parameter-to-value](~/samples/snippets/fsharp/compiler-messages/fs0003.fs#2-4)]
+
+In this snippet, the error
+
+```text
+error FS0003: This value is not a function and cannot be applied.
+```
+
+is shown because the value `v` is not a function that accepts parameters.
```

Comment:
Recommended change:

```
In the following snippet, the FS003 error is shown because the value `v` is not a function that accepts parameters. 

<error example>
```